src/VisionModel/googlenet.jl

```diff
@@ -0,0 +1,42 @@
+export googlenet
+
+function conv_block(inplanes, outplanes, stride, pad, kernel_size)
+  conv_layer = Chain(Conv(kernel_size, inplanes => outplanes, stride = stride, pad = pad),
+                     BatchNorm(outplanes, relu))
+  return conv_layer
+end
```

Comment:
<img width="74" alt="Screen Shot 2020-07-29 at 9 04 38 AM" src="https://user-images.githubusercontent.com/4811350/88810306-d2bc6a00-d17a-11ea-8ff5-bf11144bbe3a.png">

Can we have `conv_block` return an array of layers instead of a `Chain`? Then inside the `googlenet` function, you can splat the output like `Chain(conv_block(# args)..., # other layers)`. That way the resulting final chain looks like the image above (no nested `Chain`s except the inception blocks where it makes sense).

Reply:
yes